jawt-docs/dev/process.md

```
@@ -0,0 +1,256 @@
+	StatusRestarting
+	StatusFailed
+)
+```
+
+### `ProcessStats`
+
+Contains runtime statistics and information about a managed process.
+
+```go
+type ProcessStats struct {
+	Name         string        `json:"name"`
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)